sdk/core/azure-mgmt-core/sdk_packaging.toml

```diff
@@ -0,0 +1,9 @@
+[packaging]
+package_name = "azure-core"
+package_nspkg = "azure-nspkg"
+package_pprint_name = "MyService Management"
```

Comment:
Actually, this was probably copied from somewhere else.

Reply:
fixed